lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -2128,23 +2110,31 @@ namespace Js
         wchar_t *outStr = builder.DangerousGetWritableBuffer();
 
         wchar_t* outStrLim = outStr + count;
+        wchar_t *o = outStr;
+
+        while (o < outStrLim)
+        {
+            *o++ = *inStr++;
+        }
```

Comment:
No particular reason - old code just chose to use this pattern vs. memcpy() or wcscpy_s(). It can change for a good reason.